.github/workflows/pullrequests.yml

```diff
@@ -59,10 +59,10 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.7
-    - name: "Install black"
+    - name: "Install dependencies"
       run: |
         python -m pip install -U pip
-        python -m pip install black==19.10b0
+        pip install '.[dev]'
```

Comment:
i would not strip python -m here, this makes sure that the pip version that goes with that python version is being used, instead of just any random pip in the container.

Reply:
copy-pasta ftl 🤣